### PR TITLE
Bump to 1.6.0, rerender, temporary extra test mock dependency

### PR DESCRIPTION
**Pull request by bollwyvl**
Following on from #15 (thanks @sannykr!) this tries to overcome the `pytest-mock` distribution not demanding `mock` on `#[py2k]` to fix the tests.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -43,6 +44,7 @@ test:
     - pytest-mock
     - python
     - mercurial  # [py2k]
+    - mock # [py2k]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Shouldn't we just fix `pytest-mock`? It's clearly broken right?

Comment (bollwyvl) [reply recipe/meta.yaml]: Yeah, indeed. Wanted to see if this would in fact fix it...

Comment (bollwyvl) [reply recipe/meta.yaml]: PR up: https://github.com/conda-forge/pytest-mock-feedstock/pull/13

**Comment by sannykr** [comment]
@bollwyvl : Is it good to be merged?

**Comment by sannykr** [comment]
@bollwyvl : Could you please review and merge this pull request.

**Comment by bollwyvl** [comment]
@sannykr sorry for the delay: I am seeing some local test fails... will see if i can isolate what's failing...

**Comment by bollwyvl** [comment]
Looks like the local test fails were some permissions issues on my side. Just pushed with the temporary `mock  # [py27]` dependency taken out, 🤞!

**Comment by bollwyvl** [comment]
Great, it passed the first py27 test without the explicit `mock`. If all goes green, will merge...

**Comment by bollwyvl** [comment]
Thanks for sticking with this one, @sannykr!

**Pull request merged**